mlrun/runtimes/local.py

```diff
@@ -89,6 +91,11 @@ def _run(self, runobj: RunObject, execution):
             "starting local run: {} # {}".format(self.spec.command, handler or "main")
         )
 
+        if runobj.metadata.labels["kind"] == IgzSparkRuntime.kind:
+            from mlrun.runtimes.igzsparkjob import igz_spark_pre_hook
+
+            igz_spark_pre_hook()
+
```

Comment:
@urihoenig i assume we need the same for the `local` handler, in case users use py files?